src/content/en/2020/css.md

```diff
@@ -241,18 +274,20 @@ The humble `px` unit has gotten a lot of negative press over the years. At first
   sql_file="units_frequency.sql"
 ) }}
 
-The `px` unit is still going strong as the most popular length unit overall, with a whopping **72.58% of all length values across all stylesheets using `px`**! And if we exclude percentages (since they are not really a unit) the share of `px` increases even more, to 84.14%.
+{# TODO Authors—What do you think about dropping the `<` and `>` signs and code formatting here and using length instead fo `<length>`? I know that's the official syntax of CSS data types (https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Types#Syntax), but IMHO rarely used outside of CSS experts and wonder if the average reader would get confused with HTML elements? I know I did until I looked it up. Or would you prefer to keep them in here since they are actually the correct way of specifying this? #}
```

Comment:
@LeaVerou, @svgeesus, @rachelandrew - thoughts on this?

Reply:
I would be fine with dropping `<>`

Reply:
I think given that we link to docs, if anyone is confused they can follow the links. I would vote against dropping the brackets but not a strong objection.

Reply:
I agree if you kept the code formatting it would be confusing. But I'm talking about dropping both the brackets and code formatting and using the word "length" rather than the type "`<length>`" because, as I say, I don't think this is well known outside of the hardcode CSS spec contributors like yourselves, and for the casual reader it's too easily confused with HTML elements. The fact we only use it on this one type and nowhere else in this document kinda proves that point IMHO.

Reply:
OK let's close this and leave it as is. Don't think it's a biggie and was just a suggestion.

```suggestion
```